client/src/components/applicants/ApplicantModal.js

```diff
@@ -1,9 +1,8 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
-import { Modal, Button } from '@material-ui/core'
+import { Modal, Button, Chip } from '@material-ui/core'
```

Comment:
Andrea found [information about how to minimize bundle size](https://material-ui.com/guides/minimizing-bundle-size/)